src/xcode/ENA/ENA/Resources/Assets/Assets.xcassets/Certificates/EUStarsGroup.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "lightStarsNew.pdf",
+      "filename" : "Stars_half.pdf",
```

Comment:
Why you changed the stars image? Does this fix the scrolling?
Otherwise: What fixed the scrolling you mentioned in the description?

Reply:
The issue was the extra stars were appearing on the navigation bar during the scroll and this was the easiest solution possible. UX is onboard on this one.

Reply:
okay I think I got it.